__mocks__/allergies.mock.ts

```diff
@@ -1,9 +1,9 @@
-enum AllergenType {
-  DRUG = 'DRUG',
-  FOOD = 'FOOD',
-  ENVIRONMENT = 'ENVIRONMENT',
-  OTHER = 'OTHER',
-}
+const ALLERGEN_TYPES = {
+  DRUG: 'DRUG',
+  FOOD: 'FOOD',
+  ENVIRONMENT: 'ENVIRONMENT',
+  OTHER: 'OTHER',
+} as const;
```

Comment:
So we're not charlatans who don't practice what we preach 😄

![CleanShot 2025-04-04 at 1  38 52@2x](https://github.com/user-attachments/assets/0ddc8fa9-1290-4b03-ba5b-f10e95a0c393)